disabled/disabledHub.go

```diff
@@ -0,0 +1,47 @@
+package disabled
+
+import (
+	"github.com/ElrondNetwork/notifier-go/data"
+	"github.com/ElrondNetwork/notifier-go/dispatcher"
+)
+
+// Hub defines a disabled hub component
+type Hub struct{}
+
+// Run does nothing
+func (dh *Hub) Run() {
+}
+
+// BroadcastChan returns a nil channel
+func (dh *Hub) BroadcastChan() chan<- data.BlockEvents {
+	return nil
```

Comment:
is it safe to return a nil chan here? Hat happens if someone uses this as:
```GO
for _ = range dh.BroadcastChan(){

}
```

suggestion to return here a closed chan

Valid for all other occurrences in the file

Reply:
for publisher, the issue was handled based on suggestion, hub component will be updated on next task, after base components decoupling is done.
added a TODO for now

Reply:
updated hub component on this PR also